module/icp/algs/blake3/blake3_impl.c

```diff
@@ -23,10 +23,10 @@
  * Copyright (c) 2021-2022 Tino Reichardt <milky-zfs@mcmilk.de>
  */
 
+#include <sys/simd.h>
```

Comment:
Where does `blake3_impl.c` need simd.h ?

Reply:
blake3_impl.h includes simd.h and we need to load it before zfs_context.h that loads vnode.h